java/src/com/ibm/streamsx/topology/json/JSONStreams.java

```diff
@@ -144,4 +147,103 @@ public OutputTuple apply(JSONObject v1, OutputTuple v2) {
     public static <T extends JSONAble> TStream<JSONObject> toJSON(TStream<T> stream) {
         return stream.transform(new ToJSON<T>());
     }
+    
+    /**
+     * Declare a stream that flattens an array present in the input tuple.
+     * For each tuple on {@code stream} the key {@code arrayKey} is extracted
+     * and if it is an array then each value in the array will be present
+     * on the returned stream as an individual tuple.
+     * <BR>
+     * If an array value is a JSON object it will be placed on returned stream,
+     * otherwise a JSON object will be placed on the returned stream with
+     * the key {@link #PAYLOAD} containing the value.
```

Comment:
might be helpful to enhance the example below to demonstrate the PAYLOAD case too.